### ISSUE TITLE
BUG: Telegram Reaction Emojis not displayed in 100eyes request thread as answer

### ISSUE THREAD

**Issue by FrauCsu** [tactilenews/100eyes#1383]
As an editor I want to be able 
to see people's reactions to my requests, even if they are only a little emoji attached to my requests. Currently 100eyes does not display those reactions when send via Telegram.

**Comment by FrauCsu** [comment]
We already handle this with Signal. But not with Telegram so far. Or do we, @tillprochaska ?

**Comment by FrauCsu** [comment]
Hi @tillprochaska @mattwr18 it seems like we now have the same problem, see above, with Twilio / Whatsapp. Can you solve this @mattwr18? Reaction emojis via Twilio are not displayed on our instances

**Comment by mattwr18** [comment]
@FrauCsu @tillprochaska Telegram Bot's do not have access to reactions, it seems. See:
[Add method getMessageReactionsList #185](https://github.com/telegram-bot-rb/telegram-bot/issues/185).
[My Telegram bot doesn't receive emoji message reactions](https://stackoverflow.com/questions/75634281/my-telegram-bot-doesnt-receive-emoji-message-reactions)

For Twilio / WhatsApp, I just discussed this this morning with Yvonne and we receive a message when the contributor reacts however it is blank and does not contain any information that we can use to display the emoji.

For Threema, we also are not able to handle reactions.

Signal is the lone exception. We do receive reactions as text and therefore can display them the same as if the person were to respond with an emoji.

**Comment by FrauCsu** [comment]
It means we cannot do anything about it?

**Comment by mattwr18** [comment]
We could try to let the maintainers know that we are also interested in having the feature added. Other than that, I don't think we can technically do anything until it is implemented upstream.

@tillprochaska can you think of anything?

**Comment by tillprochaska** [comment]
@mattwr18 Threema only has thumbs up/down reactions as far as I know and if I remember correctly they are implemented as delivery receipt messages.

`threemarb` is still missing support for delivery receipts though, so this would have to be implemented first: https://github.com/threemarb/threema/blob/master/lib/threema/receive/delivery_receipt.rb

**Comment by mattwr18** [comment]
@tillprochaska yes, I had that in the back of my mind. Threema is different because we are maintainers of `threemarb` and so could prioritize adding support for it. I was not aware that they treat them as delivery receipts, but looking at the [documentation](https://gateway.threema.ch/en/developer/e2e#m:types:delivery-receipt), that seems plausible.

I would assume that below is referring to the thumbs up/thumbs down:

```
0x03: Message has been explicitly acknowledged.
0x04: Message has been explicitly declined.
```
